src/phoenix/trace/llama_index/callback.py

```diff
@@ -78,6 +89,10 @@ class CBEventData(TypedDict, total=False):
     attributes: Dict[str, Any]
 
 
+TraceMap = Dict[CBEventID, List[CBEventID]]
+EventMap = Dict[CBEventID, CBEventData]
```

Comment:
I think the following names would be more descriptive. (Maps are just lists with non-integer indices.)
```suggestion
ChildEventIds = Dict[CBEventID, List[CBEventID]]
EventData = Dict[CBEventID, CBEventData]
```

Reply:
addressed in [4dd7df3](https://github.com/Arize-ai/phoenix/pull/1449/commits/4dd7df31606bcffbc8c7c42456646624351b44c2)